src/backend/distributed/sql/citus--13.1-1--13.2-1.sql

```diff
@@ -1,3 +1,72 @@
 -- citus--13.1-1--13.2-1
 -- bump version to 13.2-1
 #include "udfs/worker_last_saved_explain_analyze/13.2-1.sql"
+
+#include "udfs/citus_finish_pg_upgrade/13.2-1.sql"
+
+DO $drop_leftover_old_columnar_objects$
+BEGIN
+  -- If old columnar exists, i.e., the columnar access method that we had before Citus 11.1,
```

Comment:
What about the objects from `citus_columnar` with no user tables, shouldn't we clean them as well or a drop extension `citus_columnar` will do?